Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/AppCompat/Application.cs

```diff
@@ -53,5 +49,28 @@ public static bool GetSendAppearingEventOnResume(this IPlatformElementConfigurat
 			SetSendAppearingEventOnResume(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty CanShowKeyboardOnResumeProperty = BindableProperty.Create(nameof(CanShowKeyboardOnResume), typeof(bool), typeof(Application), false);
```

Comment:
Semantically speaking here, with this feature available, the keyboard _can_ have the ability to be shown, but ultimately is or isn't, so I'd name this `ShouldShowKeyboardOnResume` / `ShouldShowKeyboardOnResumeProperty`.